dataactvalidator/config/awardProcurementFields.csv

```diff
@@ -1,194 +1,194 @@
-hospital_flag,hospital_flag,FALSE,str,1,,
-veterinary_hospital,veterinary_hospital,FALSE,str,1,,
-hispanic_servicing_institution,hispanic_servicing_institu,FALSE,str,1,,
-foundation,foundation,FALSE,str,1,,
-woman_owned_business,woman_owned_business,FALSE,str,1,,
-minority_owned_business,minority_owned_business,FALSE,str,1,,
-women_owned_small_business,women_owned_small_business,FALSE,str,1,,
-economically_disadvantaged_women_owned_small_business,economically_disadvantaged,FALSE,str,1,,
-joint_venture_women_owned_small_business,joint_venture_women_owned,FALSE,str,1,,
-joint_venture_economically_disadvantaged_women_owned_small_business,joint_venture_economically,FALSE,str,1,,
-veteran_owned_business,veteran_owned_business,FALSE,str,1,,
-service_disabled_veteran_owned_business,service_disabled_veteran_o,FALSE,str,1,,
-contracts,contracts,FALSE,str,1,,
-grants,grants,FALSE,str,1,,
-receives_contracts_and_grants,receives_contracts_and_gra,FALSE,str,1,,
-airport_authority,airport_authority,FALSE,str,1,,
-council_of_governments,council_of_governments,FALSE,str,1,,
-housing_authorities_public_tribal,housing_authorities_public,FALSE,str,1,,
-interstate_entity,interstate_entity,FALSE,str,1,,
-planning_commission,planning_commission,FALSE,str,1,,
-port_authority,port_authority,FALSE,str,1,,
-transit_authority,transit_authority,FALSE,str,1,,
-subchapter_s_corporation,subchapter_s_corporation,FALSE,str,1,,
-limited_liability_corporation,limited_liability_corporat,FALSE,str,1,,
-foreign_owned_and_located,foreign_owned_and_located,FALSE,str,1,,
-american_indian_owned_business,american_indian_owned_busi,FALSE,str,1,,
-alaskan_native_owned_corporation_or_firm,alaskan_native_owned_corpo,FALSE,str,1,,
-indian_tribe_federally_recognized,indian_tribe_federally_rec,FALSE,str,1,,
-native_hawaiian_owned_business,native_hawaiian_owned_busi,FALSE,str,1,,
-tribally_owned_business,tribally_owned_business,FALSE,str,1,,
-asian_pacific_american_owned_business,asian_pacific_american_own,FALSE,str,1,,
-black_american_owned_business,black_american_owned_busin,FALSE,str,1,,
-hispanic_american_owned_business,hispanic_american_owned_bu,FALSE,str,1,,
-native_american_owned_business,native_american_owned_busi,FALSE,str,1,,
-subcontinent_asian_asian__indian_american_owned_business,subcontinent_asian_asian_i,FALSE,str,1,,
-other_minority_owned_business,other_minority_owned_busin,FALSE,str,1,,
-for_profit_organization,for_profit_organization,FALSE,str,1,,
-nonprofit_organization,nonprofit_organization,FALSE,str,1,,
-other_not_for_profit_organization,other_not_for_profit_organ,FALSE,str,1,,
-u.s._local_government,us_local_government,FALSE,str,1,,
-referenced_idv_modification_number,referenced_idv_modificatio,FALSE,str,25,,
-undefinitized_action,undefinitized_action,FALSE,str,1,,
-domestic_or_foreign_entity,domestic_or_foreign_entity,FALSE,str,1,,
+piid,piid,FALSE,str,,,
+awardingsubtieragencycode,awarding_sub_tier_agency_c,FALSE,str,,,
```

Comment:
Yes, this is the same change that was made to D2, this makes all fields optional strings with no length limits.  Ideally an updated version of the IDD would include these changes as well.  Alternatively, we could refactor schemaLoader to overwrite these values for award_financial_assistance and award_procurement with (false, str, None) as it loads each field.  I'll add a story to JIRA proposing that change.